blockchain/fundamentals/building-the-ledger/malicious-nodes.md

```diff
@@ -22,21 +27,26 @@ The double-spend attack isn't the only threat to a blockchain and is far from be
 
 A single entity controlling more than half of the network resources is also controlling the network as a whole. Blocks can be rewritten, accounts can be emptied, transactions can be ignored. This is probably the most destructive attack that could happen.
 
-Possible in theory, but very hard in practice: the size of the network is proportional with the difficulty of pulling of such an attack. But once this happens, the attack succeeds and cannot be reversed.
+Possible in theory, but very hard in practice. The size of the network is proportional with the difficulty of pulling of such an attack. But, once this happens, the attack succeeds and cannot be reversed.
 
 ### Sybil attacks
 
 An attacker fills the network with nodes whose goal isn't to control the network per se, but rather help the execution of other attacks. If there are malicious nodes in the network, they can ignore certain transactions, allowing double-spending attacks to happen and even isolate people from the network altogether (DoS attack).  
 
 ### Replay attacks
 
-A subtype of man-in-the-middle attack, where the transaction packet sent over the network is intercepted and replayed after a certain delay. With no prevention in place, an account can be drained by repeatedly sending copies of same transaction to the nodes.
+This type of attack happens when two forked crypto-currencies allow transactions to be valid across both chains.
+
+For example, you stored all you `Coin` in one wallet. After a while, `Coin` split into `Coin1` and `Coin2`.
+
+You create a transaction saying send one `Coin` to an address `X`. You broadcast this to the `Coin1` network and, after confirmation, your transfer is successfull.
+
+In the meantime, someone sees your transaction data and copies it. This time though, they broadcast it to the `Coin2` network. If there is no replay protection, the transaction is considered valid. 
```

Comment:
Idk where you found this, but im not 100% this is accurate? every fork means a new network, with a new set of validators/miners working on that chain.

Reply:
[source](https://support.exodus.io/article/168-what-is-a-replay-attack#)

Reply:
Replay attack isn't specific to blockchains or distributed networks: it refers to a malicious agent intercepting a request that contains identifying information over an insecure network (be it wallet transactions, card payments, logins etc.) and sending it again to the same destination, pretending to be the initial sender.

As such, in the case of wallet transactions, an account can be drained by repeatedly sending copies of same transaction to the nodes.

Reply:
ok gotcha. maybe combine my explanation with yours?
```
...
As for blockchain, it might be possible to replay transactions on forked networks, where the core code/logic is the same. With no replay protection in place, a transaction broadcasted on the fork can be copy-pasted and broadcasted on the main as well, draining that account.
```